project/profiles/capacity4more/behat/features/bootstrap/FeatureContext/Group.php

```diff
@@ -698,4 +698,17 @@ public function iResetOrderToAlphabetical() {
     $el = $page->find('xpath', '//a[contains(text(),\'Order items alphabetically\') and not(ancestor::*[contains(@style,\'visibility: hidden\')])]');
     $el->click();
   }
+
+  /**
+   * @When /^I click "(?P<path>[^"]*)" under ul "(?P<field>[^"]*)"$/
+   */
+  public function assertClickUnderUl($link, $ul) {
```

Comment:
It would but I couldn't target the correct link. There is another link on the page which contains the word `Edit` and Behat follows the wrong one.